src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -496,15 +500,34 @@ var DynamicTable = new Class({
         this.selectedRows.empty();
     },
 
-    selectRow: function(rowId) {
-        this.deselectAll();
+    selectRow: function(rowId, clear) {
```

Comment:
IMO let the caller make an explicit `deselectAll()` is better than having a `clear` parameter in this function.

Reply:
I went back and forth between these two approaches. It's more explicit now.